W-15467525 xml sdk j17 update (#240)
mulesoft/docs-general: modules/ROOT/pages/_partials/partner-and-customer-guide.adoc

</dependency>
----

> . For Java SDK, add the `@JavaVersionSupport` annotation in the same class as the `@Extension` annotation and include the JAVA_17 value, for example: 

Reviewer: ```suggestion
. For Java SDK, add the `@JavaVersionSupport` annotation in the same class as the `@Extension` annotation and include the `JAVA_17` value, for example: 
```

+

> NOTE: You do not need to add any annotations for XML SDK because XML SDK modules are Java 17 compatible and will inherit the property automatically.

Reviewer: ```suggestion
NOTE: You don't need to add any annotations for XML SDK because XML SDK modules are Java 17 compatible and inherit the property automatically.
```
